### PR TITLE
Switch to onprem semaphore

### PR DESCRIPTION
**Pull request by brianstrauch**

### CODE REVIEW

File: render.go [hunk 1]
```diff
@@ -203,7 +197,6 @@ func (r *Render) Render(buffer *Buffer, previousText string, lastKeyStroke Key,
 	_, y := r.toPos(cursorEndPos)
 	h := y + 1 + int(completion.max)
 	if h > int(r.row) || completionMargin > int(r.col) {
-		//TODO: comment back in after demo
```
Comment (brianstrauch) [inline render.go:206]: @YannickPferr can this be commented back in?

Comment (YannickPferr) [reply render.go:206]: Hmm I don't think adding this behavior back in makes sense atm, it might be better to just remove this whole if block for now

Comment (YannickPferr) [reply render.go:206]: We have a ticket for addressing the problem this causes here: https://confluentinc.atlassian.net/browse/KFS-666

Comment (brianstrauch) [reply render.go:206]: Got it, this is causing a linter error since the branch is empty, so I'll temporarily remove it.

**Review by sgagniere — APPROVED** [review]

**Pull request merged**